openssl/src/ssl/mod.rs

```diff
@@ -690,6 +691,24 @@ impl Ssl {
         Ok(ssl)
     }
 
+    pub fn get_state_string<'a>(&self) -> &'a str {
+        let state = unsafe {
```

Comment:
What is the lifetime of the strings returned by these methods? If it's static, returning a `&'static str` is a bit more clear than an unbound lifetime.